projects/vertical-components/src/sidenav/sidenav-item/sidenav-item.ts

```diff
@@ -0,0 +1,66 @@
+import { AfterContentInit, ChangeDetectionStrategy, ChangeDetectorRef, Component, ContentChildren, EventEmitter, HostListener, Input, QueryList } from '@angular/core';
+
+@Component({
+  selector: 'vertical-sidenav-item',

```

Comment:
Wäre es nicht schön mit einem Selector zu arbeiten? Dann ist es auch möglich einen Anchor als SidenavItem zu verwenden, der vielleicht auf eine externe Seite in einem neuen Tab verweist

Reply:
Daran hatte ich irgendwie gar nicht gedacht. Aber habe es mal angepasst.